### ISSUE TITLE
Nylas Update 2.0.13 Broke Opening Emails from Activity Window

### ISSUE THREAD

**Issue by thebearjew** [nylas/nylas-mail#3387]
##### Are there any related issues?
Not currently

##### What operating system are you using?
macOS Sierra 10.12.4

##### What version of Nylas Mail are you using?
2.0.13

--

**Bug?**

##### Summary
This bug happens when clicking on an event in the activity window (when an email is opened or a link is clicked).

The email is not rendered but a loading animation is exposed indefinitely.

Dev tools errors:

```js
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101 TypeError: Cannot read property 'usesLabels' of undefined(…) Objectmodule.exports.ErrorLogger.reportError @ /Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/components/mail-label-set.js:59 Uncaught TypeError: Cannot read property 'usesLabels' of undefined
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101 TypeError: Cannot read property 'usesLabels' of undefined(…) Objectmodule.exports.ErrorLogger.reportError @ /Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101
/Applications/Nylas Mail.app/Contents/Resources/app.asar/node_modules/rx-lite/rx.lite.js:81 Uncaught TypeError: Cannot read property 'usesLabels' of undefined
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101 TypeError: Cannot read property 'usesLabels' of undefined(…) Objectmodule.exports.ErrorLogger.reportError @ /Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/components/mail-label-set.js:59 Uncaught TypeError: Cannot read property 'usesLabels' of undefined
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101 TypeError: Cannot read property 'disconnect' of undefined(…) Objectmodule.exports.ErrorLogger.reportError @ /Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101
/Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101 CustomEvent Objectmodule.exports.ErrorLogger.reportError @ /Applications/Nylas Mail.app/Contents/Resources/app.asar/src/error-logger.js:101
```

##### Do you have any third-party plugins installed? If so, which ones?
No third party plugins. Only third party theme (but uninstalled and confirmed the issue still exists)

##### Is the issue reproducible with a particular attachment, message, signature, etc?
See above

**Comment by jstejada** [comment]
Hi! this should be fixed in 2.0.14 which was released today. Let me know if that works!

thanks!

**Comment by thebearjew** [comment]
@jstejada Thanks for the quick response! I'll keep an eye out!
